demo-shell/src/app/components/files/files.component.scss

```diff
@@ -76,28 +76,10 @@
 
     .adf-manage-versions-sidebar {
         width: 360px !important;
-        color: rgba(0, 0, 0, 0.87);
```

Comment:
Why do we need to remove this part?

Reply:
:) I tested how it looks after I remove it and there is no difference with or without the 'color: rgba(0, 0, 0, 0.87);' line.
Maybe it would be good to have a task to clean-up the demo-shell scss files and also move styles from there to components...

Reply:
@eromano it's a demo shell file, always nice to clean it a bit as for me :)